core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -725,7 +727,11 @@ object KafkaConfig {
 
   /************* Authorizer Configuration ***********/
   val AuthorizerClassNameDoc = s"The fully qualified name of a class that implements <code>${classOf[Authorizer].getName}</code>" +
-  " interface, which is used by the broker for authorization."
+    " interface, which is used by the broker for authorization."
+  val EarlyStartListenersDoc = "A comma-separated list of listener names which should be started before any non-early start listeners. " +
```

Comment:
> nit: the first sentence is seems circular

Hmm, I don't know -- "started before any non-early-start listeners" does seem to convey some information. Circular would be more like "early start listeners are early start listeners," right? :) Maybe I'm misunderstanding your point.

> I think it would be helpful to mention the authorizer use case.

The second sentence says "This is useful in cases where the startup process requires some listeners to be open before other listeners can be brought up" which I thought was stating the use-case... do you have an idea for improving it?

> Also, can we say something about the default?

Good idea. I added a bit about the default (of controller listeners) at the end.

Reply:
The circularity was that we seemed to be defining the early start listeners in terms of the non-early start listeners. Here is my suggestion:

> A comma-separated list of listener names which should be started before the authorizer has finished initialization. This is useful when the authorizer is dependent on the cluster itself for bootstrapping, as is the case for the standard authorizer which stores ACLs in the metadata log. By default, all listeners included in `controller.listener.names` will also be early start listeners.

Reply:
OK. I took your text as the doc for early.start.listeners (with some minor changes)